substrate/frame/ranked-collective/src/lib.rs

```diff
@@ -365,6 +368,13 @@ impl_ensure_origin_with_arg_ignoring_arg! {
 	{}
 }
 
+/// Helper functions to setup benchmarking.
+#[impl_trait_for_tuples::impl_for_tuples(8)]
+pub trait BenchmarkSetup<AccountId> {
+	/// Ensure that this member is registered correctly.
+	fn ensure_member(acc: &AccountId);
```

Comment:
nit: I'd rename this to something like `register_as_member` or similar because you're not really running checks and returning the validity, you're doing setup work.

Reply:
Yea true. But will do it as follow up to not retrigger the CI now to not prevent the merge.